feconf.py

```diff
@@ -132,6 +132,9 @@
 ENTITY_TYPE_QUESTION = 'question'
 ENTITY_TYPE_VOICEOVER_APPLICATION = 'voiceover_application'
 
+VOICEOVER_APPLICATION_REVIEW = 'review'
```

Comment:
Constants are usually named as "constant_type"_"label_for_specific_constant". In this case it's not clear what the constant type is.

Reply:
Removes as these constants are not required anymore!